### PR TITLE
[INTER-535] Revisit angular repo

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ The following dependencies are required:
 
 - TypeScript >=4.6
 - Node 16+
-- Angular 13+
+- Angular 15+
 
 ## Installation
```
Comment (JuroUhlar) [inline README.md:57]: @TheUnderScorer Could you add section on linkedIds as part of this PR? https://fingerprintjs.atlassian.net/browse/INTER-586

File: tsconfig.json [hunk 2]
```diff
@@ -23,7 +24,7 @@
     "moduleResolution": "node",
     "importHelpers": true,
     "resolveJsonModule": true,
```
[the remark this answers is not in the document]
Comment (TheUnderScorer) [reply tsconfig.json:25]: Done!

File: package.json [hunk 3]
```diff
@@ -58,17 +64,17 @@
     "eslint-plugin-prettier": "^4.0.0",
```
[the remark this answers is not in the document]
Comment (TheUnderScorer) [reply package.json]: Done!

File: package.json [hunk 4]
```diff
@@ -15,41 +15,47 @@
...
+    "@angular/animations": "^15.2.10",
+    "@angular/common": "^15.2.10",
+    "@angular/compiler": "^15.2.10",
+    "@angular/core": "^15.2.10",
+    "@angular/forms": "^15.2.10",
+    "@angular/platform-browser": "^15.2.10",
+    "@angular/platform-browser-dynamic": "^15.2.10",
+    "@angular/platform-server": "^15.2.10",
+    "@angular/router": "^15.2.10",
+    "@fingerprintjs/fingerprintjs-pro-spa": "^1.2.0",
```
[the remark this answers is not in the document]
Comment (TheUnderScorer) [reply package.json]: Good point, done!

File: package.json [hunk 5]
```diff
@@ -15,41 +15,47 @@
...
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~13.3.6",
-    "@angular-eslint/schematics": "^13.2.1",
-    "@angular/cli": "~13.3.6",
-    "@angular/compiler-cli": "~13.3.0",
+    "@angular-devkit/build-angular": "^15.2.10",
+    "@angular-eslint/schematics": "^15.2.1",
+    "@angular/cli": "^15.2.10",
```
[the remark this answers is not in the document]
Comment (TheUnderScorer) [reply package.json:44]: By `ng update`

File: README.md (hunk 1 above)
Comment (TheUnderScorer) [reply README.md:57]: Yep, done!

File: package.json [hunk 6]
```diff
@@ -15,60 +15,61 @@
...
+    "@types/jest": "^29.5.12",
     "@types/node": "^12.11.1",
-    "@typescript-eslint/eslint-plugin": "^5.27.0",
-    "@typescript-eslint/parser": "^5.27.0",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
+    "@typescript-eslint/parser": "^6.21.0",
     "commitizen": "^4.2.5",
-    "conventional-changelog-conventionalcommits": "^5.0.0",
     "cz-conventional-changelog": "^3.3.0",
     "eslint": "^8.16.0",
```
Comment (ilfa) [inline package.json:59]: `eslint` is a dependency of a `"@fingerprintjs/eslint-config-dx-team"` too to simplify version update and keep all packages compatible

**Review by ilfa — DISMISSED** [review]

**Review by ilfa — APPROVED** [review]

**Pull request merged**